advanced/is-pattern-2/requirements.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@darsham3 why have we added the truststore along with the Helm resources? Have we customized the default truststore we get in the product pack?

Reply:
@chirangaalwis default trust store does not contain the public certificate of analytics.so mounted updated trust store with analytic certificate.

Reply:
+1